docker-assets/initialize-httpd-auth.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+HTTPD_AUTH_CONFIG_DIR=/etc/httpd/auth-conf.d
+AUTH_CONFIG_FILE="${HTTPD_AUTH_CONFIG_DIR}/auth-configuration.conf"
+SSSD_CONF_FILE="/etc/sssd/sssd.conf"
```

Comment:
@abellotti @carbonin Could we put the initialization scripts on their own systemd service unit like we use to do in fine : 

https://github.com/ManageIQ/manageiq-pods/blob/fine/images/miq-app/docker-assets/appliance-initialize.service

This will allow to free ENTRYPOINT and support OCI systemd hooks properly.